packages/web-react/src/components/Select/stories/ChildrenNode.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 
-export default () => (
-  <>
-    <option value="1">Option 1</option>
-    <option value="2">Option 2</option>
-    <optgroup label="Sub options">
-      <option value="3">Option 3</option>
-      <option value="4">Option 4</option>
-    </optgroup>
-  </>
-);
+export default (props: { isDisabledPlaceholder?: boolean }) => {
```

Comment:
is this in the right commit?

Reply:
Whats the problem here?

Reply:
This is added in commit [Chore(web-react): Add Vite React plugin to demo app configuration](https://github.com/lmc-eu/spirit-design-system/pull/868/commits/346fa8ac431d3672699bb08758c5586ef5f3366c) where the commit message doesn't really relate to Select. So maybe this should be in another commit? But I will let @literat decide.

Reply:
This whole conversion is meant for the whole file, sorry I placed it somewhere in the middle, which maybe made it confusing.

Reply:
Please ask @literat about this commit, thank you

Reply:
I think that @tomassychra used the wrong target for the fixup commit. Please be more careful when choosing to where place the fixup. Otherwise, this is already rebased.

Reply:
I have manually rebased this.